sympy/printing/__init__.py

```diff
@@ -10,13 +10,18 @@
 
 from .pycode import pycode
 
-from .ccode import ccode, print_ccode
+from .codeprinter import print_ccode, print_fcode
 
-from .glsl import glsl_code, print_glsl
-
-from .cxxcode import cxxcode
+# We have to import these now because otherwise if there are imported later,
+# they will override sympy.printing.ccode and so on with the
+# modules. It would be nice to not have to do this as we could avoid importing
+# these files at top-level import time completely otherwise.
```

Comment:
Ah, of course we can't do this, because this makes sympy.codegen be imported again, which defeats the whole purpose. So I think the only fix here is to rename the files.